libnavui-androidauto/src/main/java/com/mapbox/androidauto/map/MapboxCarMapLoader.kt

```diff
@@ -65,6 +65,17 @@ class MapboxCarMapLoader : MapboxCarMapObserver {
         mapboxMap = null
     }
 
+    /**
+     * Returns the current style contract. If an override has not been set the default is returned.
+     */
+    fun getCurrentExtension(isDarkMode: Boolean): StyleContract.StyleExtension {
```

Comment:
Should we pass `CarContext` here? Or pass `isDarkMode` in `onCarConfigurationChanged`?

Reply:
Considering a use for it to be, updating a `StyleContract` while it is not set (while the car is in dark mode you can still access the light mode contract)